src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services10.php

```diff
@@ -30,6 +30,11 @@ public function __construct()
         $this->parameters = $this->getDefaultParameters();
 
         $this->services = array();
+        $this->normalizedIds = array(
+            'psr\\container\\containerinterface' => 'Psr\\Container\\ContainerInterface',
```

Comment:
dunno :) do we really care? if someone wants to try fixing that, please do :) should be done while dumping - ie in PhpDumper I'd say.